composer.json

```diff
@@ -44,6 +44,10 @@
         "sort-packages": true,
         "platform": {
             "php": "7.4.7"
+        },
+        "allow-plugins": {
```

Comment:
I added the `allow-plugins` property, since it will be a required property in the near future. See https://getcomposer.org/doc/06-config.md#allow-plugins